.github/workflows/update-intellij-version.yml

```
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+      - name: Prepare variables for PR
+        # language=sh
+        run: |
+          release_line=$(grep 'release = \"' core/driver/sources/src/main/resources/reference.conf)
+          intellij_version=$(awk -F'\"|\"' '{print $2}' <<< "$release_line")
+          echo "BRANCH_NAME=update_intellij_version_to_$intellij_version" >> $GITHUB_ENV
```

What if a branch of this name (and/or a PR for such branch) already exists? 🤔  See the use of `git ls-remote` in git-machete-intellij-plugin

---

Also, nit:
```suggestion
          echo "BRANCH_NAME=update_intellij_version/$intellij_version" >> $GITHUB_ENV
```
The `general_prefix/specific_thing` is a well-established convention for git branch names

---

- if a branch already exists and no PR for this branch - then `peter-evans/create-pull-request@v4` will add another commit on existing branch and create a pull request
- if a branch exists and there is a PR for this branch - then `peter-evans/create-pull-request@v4` will add another commit on existing branch on the same PR

---

`BRANCH_NAME` changed as suggested